front/app/components/UI/FullscreenModal/index.tsx

```diff
@@ -147,7 +161,34 @@ const FullscreenModal = ({
         contentBgColor={contentBgColor}
       >
         <StyledFocusOn autoFocus>
-          {topBar}
+          {modalTitle && (
+            <Box
+              bgColor={colors.white}
+              borderBottom={`1px solid ${colors.grey300}`}
+              display="flex"
+              alignItems="center"
+              justifyContent="center"
```

Comment:
Centering the content, which is why close icon is taken out of the normal layout flow below.